### ISSUE TITLE
Tone.Loop Constructor does not set humanize

### ISSUE THREAD

**Issue by nmarshall23** [Tonejs/Tone.js#1149]
**Describe the bug**

Tone.Loop does not set humanize option when using the options Constructor.

[On line 55](https://github.com/Tonejs/Tone.js/blob/092fc690f8f4fd858dc0fef2cfc4be2446e303c1/Tone/event/Loop.ts#L51)
When creating a ToneEvent it should set humanize.

**To Reproduce**

```
const loop = new Tone.Loop({
	callback: (time) => console.log(time),
	interval: 4,
	humanize: 16,
}).start(0);
console.log(loop.humanize) // this is false
Tone.Transport.start();
```

**Comment by pritam-chowdhury** [comment]
@tambien @nmarshall23  I have raised a pull request((1154) for the issue .
can you please have a look and revert back to it
Thanks :)

**Comment by tambien** [comment]
fixed by #1154
